apps/nsqd/main.go

```diff
@@ -72,7 +62,15 @@ func (p *program) Start() error {
 	}
 	p.nsqd = nsqd
 
-	err = p.nsqd.LoadMetadata()
+	if env.IsWindowsService() {
+		dir := filepath.Dir(os.Args[0])
+		return os.Chdir(dir)
```

Comment:
This used to be before `nsqd.New()` (which uses `os.Getwd()` for default datapath), also before opening the config file ... this only affects windows services, but still might want to keep their behavior consistent with before, I dunno.

Reply:
Good catch